security.php

```diff
@@ -97,9 +97,14 @@ function wpcom_vip_login_limit_dont_show_login_form() {
 }
 add_action( 'login_form_login', 'wpcom_vip_login_limit_dont_show_login_form' );
 
+
 function wpcom_vip_login_limit_xmlrpc_error( $error, $user ) {
-	if ( is_wp_error( $user ) && ERROR_CODE_LOGIN_LIMIT_EXCEEDED == $user->get_error_code() )
+	static $login_limit_error;
+	if ( ( is_wp_error( $user ) && ERROR_CODE_LOGIN_LIMIT_EXCEEDED === $user->get_error_code() ) || ERROR_CODE_LOGIN_LIMIT_EXCEEDED === $login_limit_error ) {
+		// We need to set a persistent error here, as once there is an auth error in a system.multicall, core will no longer trigger any of the rate limit filters for further login attempts in the set.
+		$login_limit_error = ERROR_CODE_LOGIN_LIMIT_EXCEEDED;
 		return new IXR_Error( 503, $user->get_error_message() );
```

Comment:
The faultString for `1+n` returns blank:

```
  <value><struct>
  <member><name>faultCode</name><value><int>503</int></value></member>
  <member><name>faultString</name><value><string>You have exceeded the login limit.  Please wait a few minutes and try again.</string></value></member>
</struct></value>
  <value><struct>
  <member><name>faultCode</name><value><int>503</int></value></member>
  <member><name>faultString</name><value><string></string></value></member>
</struct></value>
```

Reply:
Good catch. Changed the logic to persist the error across calls so that we have access to the message.